src/web/templates/search.html

```diff
@@ -22,7 +22,7 @@ <h1 class="firstHeading">{{ gettext('Results of the search for') }} <i>'{{ searc
   <ul>
   {% for link, title, _, tokens, text in results %}
     <li class="search-result">
-        <div class="search-result-title"><a href="/{{ link }}">{{ title }}</a></div>
+        <div class="search-result-title"><a href=/{{ link }}>{{ title }}</a></div>
```

Comment:
Pero no deberían todos estos modificadores estar entre comillas?

Creo que lo correcto sería "quotear" correctamente a `link`, no?

Reply:
Si! estuve buscando y pensé que venia por ese lado pero me parece que el error viene de otro lado posiblemente de jinja o del scrip js la info hasta el html llega bien.
El tema pasa cuando lo extrae el `for` del template.
Cuando se genera toma el valor de tokens en link  o sea, fusiona link con tokens.
La solucion mas rapida y que funciona bien fue esta. 
Por ahi hay algo que no vi en el medio.
Me fijo entonces si puedo escaparlas de alguna manera antes para que pase bien en el for y llegue bien el link asi dejamos las comillas.

Reply:
Creo que "funciona bien" no es sinonimo de que esté bien, si no tiene las comillas por afuera pero creo que no es html válido y browsers mas antiguos (o mas nuevos) podrian no andar. Acá hay un link a un html validator que hace mucho usaba: https://validator.w3.org/#validate_by_input como para probar a ver si es valido o no

Reply:
Lo del quoting que dice facu es que html valido "escapea" de una forma especial las comillas (y otros caracteres especiales como el & ) usando una tecnica especifica que se logra usando `urllib.parse.quote` que convierte la comillas en `%22`:
>>> urllib.parse.quote('"foo')
'%22foo'

Reply:
Y para mí hay que hacer ese escapeo, porque hoy te lo encontraste con las comillas, mañana puede ser con otra cosa... no?

Reply:
> Creo que "funciona bien" no es sinonimo de que esté bien, si no tiene las comillas por afuera pero creo que no es html válido y browsers mas antiguos (o mas nuevos) podrian no andar. Acá hay un link a un html validator que hace mucho usaba: https://validator.w3.org/#validate_by_input como para probar a ver si es valido o no

A genial lo voy a probar

Reply:
> Lo del quoting que dice facu es que html valido "escapea" de una forma especial las comillas (y otros caracteres especiales como el & ) usando una tecnica especifica que se logra usando `urllib.parse.quote` que convierte la comillas en `%22`:
> 
> ```python
> >>> urllib.parse.quote('"foo')
> '%22foo'
> ```

Gracias por el dato no lo habia hecho, esto lo probe pero el problema se da igual.

`urllib.quote(link.encode('utf8'))`

que devuelve el link quoteado en python 2
consulta para que estariamos usando el tokens en el template?
voy a seguir haciendo pruebas.

Reply:
> Y para mí hay que hacer ese escapeo, porque hoy te lo encontraste con las comillas, mañana puede ser con otra cosa... no?

Dale lo agrego, otra cosa que vi y probé es que en:

` on_search_results` de web_app.py

 Esta repetido el codigo de un metodo de Searcher:

` get_grouped`

Esto lo saque y lo probe directamente con este metodo y funciona bien. Esto no fixea el problema de la url.

Sigo buscando.

Reply:
> > Lo del quoting que dice facu es que html valido "escapea" de una forma especial las comillas (y otros caracteres especiales como el & ) usando una tecnica especifica que se logra usando `urllib.parse.quote` que convierte la comillas en `%22`:
> > ```python
> > >>> urllib.parse.quote('"foo')
> > '%22foo'
> > ```
> 
> Gracias por el dato no lo habia hecho, esto lo probe pero el problema se da igual.
> 
> `urllib.quote(link.encode('utf8'))`
> 
> que devuelve el link quoteado en python 2
> consulta para que estariamos usando el tokens en el template?
> voy a seguir haciendo pruebas.

Mil disculpas @spiccinini y @facundobatista  si funciona pero lo habia probado "in place"
```python
 >>> link = urllib.quote(link.encode('utf8'))
```
asi lo probe ahora!

Si les parece cerramos este PR y envio otro con lo cambios.